lib/praxis-mapper/query/base.rb

```diff
@@ -51,11 +67,18 @@ def connection
       # @example select(:account_id, "user_id", {"create_time" => :created_at})
       def select(*fields)
         if fields.any?
-          @select = {} if @select.nil?
+          return @select if @select == true
+
+          @select = {} if @select.nil? # FIXME: prepopulate this with identities
```

Comment:
Should probably get this in. We didn't do it before, but it's more likely to become an issue at some point.